src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,9 @@
+import NavigationLinks from './NavigationLinks';
+
+export default function Navigation() {
+	return (
+		<nav>
+			<NavigationLinks />
+		</nav>
+	);
+}
```

Comment:
Do you really need  both `Navigation.js and NavigationLinks.js` as  separate components? 
I am not sure, maybe there is a good reason and this is a "taste-thing", as well  🤔

Reply:
Yeah we need both since we want to add a Desktop View of our MobileNavigation later on, when we style it and make it responsive.